### ISSUE TITLE
Pokemon Heart Gold - evolving to Alakazam

### ISSUE THREAD

**Issue by crapdeno** [iNDS-Team/iNDS#185]
How can we evolve Kadabra to Alakazam? I read that trading system is not supported... so any other ways??

**Comment by blueingsh** [comment]
i don’t think there is a way to evolve as you need to trade which iNDS doesn’t support

**Comment by blueingsh** [comment]
also i think there isnt more ways to

**Comment by crapdeno** [comment]
I think i might have found a solution. After sycing via iTunes, ROM and saved files can be copied. Then ROM can be changed with Universal Pokemon Randomizer. After that, the changed ROM can be copied back to iPhone via iTunes with a different name, then the saved files can be copied as well after changing the file names with new ROM name.

I tried it, I was able to start the new ROM via the emulator, however the saved file got frozen :( Gonna try with a new save file. 

https://pokehacks.dabomstew.com/randomizer/
